src/test/java/org/prebid/server/auction/ExchangeServiceTest.java

```diff
@@ -310,6 +310,46 @@ public void shouldExtractMultipleRequests() {
                 .element(0).returns(2, imp -> imp.getExt().get("bidder").asInt());
     }
 
+    @Test
+    public void shouldPassRequestWithExtPrebidToDefinedBidder() {
+        // given
+        final String bidder1Name = "bidder1";
+        final String bidder2Name = "bidder2";
+        final Bidder<?> bidder1 = mock(Bidder.class);
+        final Bidder<?> bidder2 = mock(Bidder.class);
+        givenBidder(bidder1Name, bidder1, givenEmptySeatBid());
+        givenBidder(bidder2Name, bidder2, givenEmptySeatBid());
+
+        final ObjectNode ext = mapper.valueToTree(ExtBidRequest.of(
+                ExtRequestPrebid.builder()
+                        .bidders(mapper.createObjectNode()
+                                .putPOJO(bidder1Name, mapper.createObjectNode().put("test1", "test1"))
+                                .putPOJO(bidder2Name, mapper.createObjectNode().put("test2", "test2"))
+                                .putPOJO("spam", mapper.createObjectNode().put("spam", "spam")))
+                        .build()));
+
+        final BidRequest bidRequest = givenBidRequest(asList(
+                givenImp(singletonMap(bidder1Name, 1), identity()),
+                givenImp(singletonMap(bidder2Name, 2), identity())),
+               reqBuilder -> reqBuilder.ext(ext));
+
+        // when
+        exchangeService.holdAuction(givenRequestContext(bidRequest));
+
+        // then
+        final ArgumentCaptor<BidRequest> bidRequest1Captor = ArgumentCaptor.forClass(BidRequest.class);
+        verify(httpBidderRequester).requestBids(same(bidder1), bidRequest1Captor.capture(), any(), anyBoolean());
+        final BidRequest capturedBidRequest1 = bidRequest1Captor.getValue();
+        assertThat(capturedBidRequest1.getExt().get("prebid").get("bidders").fields()).hasSize(1)
```

Comment:
We should also verify that "prebid" and "bidders" exist before getting their value, applies to the next assertion as well